xo-server/src/server.ts

```diff
@@ -186,14 +186,16 @@ class Linter {
 
 				// Clean previously computed code actions.
 				delete this.codeActions[uri];
-
-				const diagnostics: Diagnostic[] = report.results[0].messages.map((problem: any) => {
-					const diagnostic = makeDiagnostic(problem);
-					this.recordCodeAction(document, diagnostic, problem);
-					return diagnostic;
-				});
-
-				this.connection.sendDiagnostics({uri, diagnostics});
+				const results = report.results;
+				if(results.length && results[0].hasOwnProperty('messages')) {
```

Comment:
`if (result.length > 0 && results[0].messages) {`

Reply:
Given that 0 is evaluated as false, is there a reason for you wanting the ' > 0' explicitly in the if statement?  I'm just curious, is it just a 'best practices' thing or...?

Reply:
Nope I don't think there is. It's just more codestyle. I'm rather more explicit about these things.

Reply:
Fair enough, I've seen both being used so just curious. :) Keep up the good work.